migration-guide/now-to-next.adoc

```diff
@@ -0,0 +1,265 @@
+
+[id=mapping-next]
+include::../partials/version-mapping-next.adoc[]
+
+[start=5]
+. Add the `IBM ILMT Compliance Library` to the `pom.xml`  _(this is required in order to report on subscription entitlement usage within the system)_:
++
+[source,xml]
+----
+77  +++   <dependency>
+78  +++     <groupId>com.ibm.bamoe</groupId>
+79  +++     <artifactId>bamoe-ilmt-compliance-quarkus-dmmoe</artifactId>
+80  +++   </dependency>
+81      </dependencies>
+----
+
+[NOTE]
+====
+You will notice that the newly added Maven dependency includes the `groupId` and `artifactId` portions of the Maven coordinates, but does not specifcy a `version` tag.
+This is due to importing the `bamoe-bom` into the project, where the various dependency versions are managed.
+====
+
+[start=6]
+. Build the project in the root folder:
++
+[source,console]
+----
+mvn clean package
+----
++
+You will get some Maven streaming text once again, and at the end of it you should have:
++
+[source,console]
+----
+[INFO] Analysing decision tables in DMN Model 'pricing' ...
+[INFO]  analysis for decision table 'Base price':
+[INFO]   Decision Table Analysis of table 'Base price' finished with no messages to be reported.
+[INFO] [io.quarkus.deployment.QuarkusAugmentor] Quarkus augmentation completed in 3012ms
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time:  16.084 s
+[INFO] Finished at: 2023-06-28T14:22:02-03:00
+[INFO] ------------------------------------------------------------------------
+----
+
+[NOTE]
+====
+The choice of extensions determines the imported behavior of the project, including adding sample models or testing related code.
+For example, by adding the 'dmn' extension not only adds the proper dependencies, but also adds a sample `.dmn` model to the project, which acts as a template to work from.
+====
+
+Once the project has been generated, you should see a project structure in VS Code, similar to the following diagram:
+
+.Example Maven Project in VS Code
+image::maven-project.png[Example Maven Project in VS Code]
+
+As you can see from the figure above, a Maven compliant project has been generated, including all of the standard folders typically found in a Maven project.
+The most significant folder is the `resources` folder, which is where new or migrated business automation asset files should be stored.
+Examples of supported business automation assets are:
+
+- Decision Model & Notation (DMN) Models (.dmn)
+- Drools Rule Language (DLR) Files (.drl)
+- Business Process Modeling & Notation (BPMN) Models (.bpmn)
+- Excel Decision Tables (XLS) Files (.xls)
+- Standard Property Files (.properties)
+- Test Scenario Files (.scesim)
+- Test Scenario Files (.scesim)
```

Comment:
@blulas @LightGuard Test Scenario is duplicated in the list. 
Please note, the correct directory for scesim file is `test\resources` because scesim is a Testing asset.